index.html

```diff
@@ -11,7 +11,7 @@
 
 <body>
 
-<div id="app">
+<!-- <div id="app">
```

Comment:
why those comments?

Reply:
I commented it to see template for starting screen. Now it is corrected.